tests/legacy-cli/e2e/tests/packages/webpack/test-app.ts

```diff
@@ -6,9 +6,9 @@ import {expectFileSizeToBeUnder, replaceInFile, expectFileToMatch} from '../../.
 
 export default function(skipCleaning: () => void) {
   return Promise.resolve()
-    .then(() => createProjectFromAsset('webpack/test-app-ng5'))
+    .then(() => createProjectFromAsset('webpack/test-app'))
     .then(() => exec(normalize('node_modules/.bin/webpack-cli')))
-    .then(() => expectFileSizeToBeUnder('dist/app.main.js', 375 * 1024))
+    .then(() => expectFileSizeToBeUnder('dist/app.main.js', 470 * 1024))
```

Comment:
Why did this get larger?

Reply:
The framework version was change from 5 to 8.